web/styles/settings.css

```diff
@@ -1860,6 +1890,34 @@ $option_title_width: 180px;
     #change_email_modal {
         width: 400px;
     }
+
+    .upload-file-name {
+        min-width: 10em;
+    }
+
+    #user_topics_table {
+        .user-topic {
+            min-width: 10em;
+        }
+
+        .user-topic-stream {
+            min-width: 6em;
+        }
+    }
+
+    .user_row {
+        .user_name,
+        .email {
+            min-width: 8em;
+        }
+    }
```

Comment:
These CSS selectors should be scoped inside some outer settings panel class; lots of places in the app might have something called `user_row` or `upload-file-name`. This seems to be a preexisting issue with this media query block, though, so not going to block merging on it.